docs/concepts.md

```diff
@@ -0,0 +1,39 @@
+# Meltano Cloud Concepts
+
+## Meltano Cloud Users
+
+Your Meltano Cloud user account is associated with a GitHub account for single-sign-on. If you do not have a GitHub account, you can create a free account at [github.com](https://github.com).
+
+> **Note**
+>
+> During the Private and Public Beta phases, we will be collecting feedback from users regarding the login and account creation experience overall. Post-GA, we plan to offer SAML as a premium feature for Enterprise users. If you have a use case to support alternate forms of login besides GitHub auth and SAML (e.g. GitLab, or user/password auth), please let your account manager know or log an issue in our [issue tracker](https://github.com/meltano/cloud-docs/issues).
+
+## Meltano Cloud Organizations
+
+Your Meltano Cloud Organization is the top-level entity associated with your payment info. Each organization will be issued a unique billing account number.
+
+> **Note**
+>
+> Meltano Cloud Organizations should not be confused with GitHub Organizations (or "GitHub Orgs"). Each Meltano Cloud Organization can include projects from multiple GitHub Orgs.
+>
+> Meltano Cloud users grant access to specific public and private repositories using the standard GitHub App authorization flow.
+
+## Meltano Cloud Projects
+
+A "Cloud Project" is any Meltano project you have registered on Meltano Cloud. The project definition consists of a git repo and a relative project directory within the git repo.
+
+## Meltano Cloud Environments (aka "Deployments")
```

Comment:
```suggestion
## Meltano Cloud Deployments
```

What do you think about just sticking with Deployment here? Mixing in environment again just feels clunky.

Reply:
Maybe "Deployed Environments" would be ideal? It's fairly clear as to what this means both to a Meltano user and a newcomer.

Reply:
Maybe... I'm still hestiant though b/c it's not just a deployed environment - it's a deployed environment with a specific git branch/sha.

Reply:
I've updated the language here to use "Deployment" more consistently. Will commit and push momentarily.